app/assets/stylesheets/pages/_show-collection.scss

```diff
@@ -32,8 +33,15 @@
   }
 
   .collection_revision_header {
-    height: 95px;
-    line-height: 90px;
+    height: 60px;
+    line-height: 60px;
+    vertical-align: middle;
+    font-size: 1em;
+  }
+
+  .granules_revision_header {
```

Comment:
these two classes are using the same stylings. I made a comment in the other PR about combining them to reduce duplication.